contributing/contributing-code.md

```diff
@@ -36,10 +36,20 @@ Significant changes (e.g. new components or significant changes to behavior) req
 
 ## Finding a reviewer for small changes
 
-If you're making a minor fix and just want to find someone to review/approve your code, look at the OWNERS.yaml files in the directories where you're modifying code and either:
-- assign your PR to one of them if you are able to
+All code must be reviewed and approved as described in the [Code review and approval](#code-review-and-approval) section.
+
+If you're making a minor fix and just want to find someone to review/approve your code, choose:
+
+- an Owner from the OWNERS.yaml file in the directories you change (or their parent directories)
+- and a [Reviewer](https://github.com/orgs/ampproject/teams/reviewers-amphtml)
```

Comment:
One "bug" in our approach here is that these teams are not, and cannot be made publicly visible.

I wonder if we should spend the time to write a quick web app that publishes the teams on the web?

Reply:
I filed https://github.com/ampproject/meta/issues/32 to track adding this.  I figure we can get this PR merged and handle that separately.

Reply:
Since the owners bot already fetches the teams information on each pull request this information can easily be exposed in the checks tab but i dont know if that is the correct place to enumerate them.

Reply:
Joey and I discussed that the best place is really to just put it on amp.dev which should be super straight forward. Part of the site is already generated from GH API calls.